src/styles/styles.ts

```diff
@@ -0,0 +1,3979 @@
+            fontSize: variables.fontSizeXXLarge,
+            lineHeight: variables.lineHeightXXLarge,
+            wordBreak: 'break-all',
+        },
+
+        eReceiptMerchant: {
+            fontFamily: fontFamily.EXP_NEUE,
+            fontSize: variables.fontSizeXLarge,
+            lineHeight: variables.lineHeightXXLarge,
+            color: theme.text,
+        },
+
+        eReceiptWaypointTitle: {
+            fontFamily: fontFamily.EXP_NEUE,
+            fontSize: variables.fontSizeSmall,
+            lineHeight: variables.lineHeightSmall,
+            color: colors.green400,
+        },
+
+        eReceiptWaypointAddress: {
+            fontFamily: fontFamily.MONOSPACE,
+            fontSize: variables.fontSizeNormal,
+            lineHeight: variables.lineHeightNormal,
+            color: theme.textColorfulBackground,
+        },
+
+        eReceiptGuaranteed: {
+            fontFamily: fontFamily.MONOSPACE,
+            fontSize: variables.fontSizeSmall,
+            lineHeight: variables.lineHeightSmall,
+            color: theme.textColorfulBackground,
+        },
+
+        eReceiptBackground: {
+            ...sizing.w100,
+            borderRadius: 20,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            height: 540,
+        },
+
+        eReceiptPanel: {
+            ...spacing.p5,
+            ...spacing.pb8,
+            ...spacing.m5,
+            backgroundColor: colors.green800,
+            borderRadius: 20,
+            width: 335,
+        },
+
+        eReceiptBackgroundThumbnail: {
+            ...sizing.w100,
+            position: 'absolute',
+            aspectRatio: 335 / 540,
+            top: 0,
+            minWidth: 217,
+        },
+
+        eReceiptContainer: {
+            flex: 1,
+            width: 335,
+            minHeight: 540,
+            borderRadius: 20,
+            overflow: 'hidden',
+        },
+
+        loginHeroBody: {
+            fontFamily: fontFamily.EXP_NEUE,
+            fontSize: variables.fontSizeSignInHeroBody,
+            color: theme.textLight,
+            textAlign: 'center',
+        },
+
+        linkPreviewWrapper: {
+            marginTop: 16,
+            borderLeftWidth: 4,
+            borderLeftColor: theme.border,
+            paddingLeft: 12,
+        },
+
+        linkPreviewImage: {
+            flex: 1,
+            resizeMode: 'contain',
+            borderRadius: 8,
+            marginTop: 8,
+        },
+
+        linkPreviewLogoImage: {
+            height: 16,
+            width: 16,
+        },
+
+        contextMenuItemPopoverMaxWidth: {
+            maxWidth: 375,
+        },
+
+        formSpaceVertical: {
+            height: 20,
+            width: 1,
+        },
+
+        taskCheckbox: {
+            height: 16,
+            width: 16,
+        },
+
+        taskTitleMenuItem: {
+            ...writingDirection.ltr,
+            ...headlineFont,
+            ...flex.flexWrap,
+            ...flex.flex1,
```

Comment:
This was `spacing.flexWrap`, `spacing.flex1` originally which didn't exist.
So let's remove these bad styles in https://github.com/Expensify/App/pull/29629 to fix https://github.com/Expensify/App/issues/29627